### DISCUSSION TITLE
users date control expiration

### DISCUSSION THREAD

**Discussion by ruiribeiro2010** [jellyfin/jellyfin#11768] [General]
Hello everybody,
congratulations on your work.

Wouldn't it be possible to add a feature so we can control users using dates?
In other words, an expiration date was defined for a specific user, etc.

compliments

**Comment by crobibero** [comment]
No, Jellyfin is a personal media server.

**Reply by ruiribeiro2010** [reply]
Yes I understand.

but we may want to add a family member or friend, and add an expiration date for them. It was a good resource and it was easier to manage.

Or, we could have a plugin for anyone who wanted to add this option, so only those who needed it could do it.

**Comment by TimGels** [comment]
Jellyfin itself does not have this functionality and it seems out of scope, at least as things are right now. But you might want to look into other authentication methods. One thing that might work is using LDAP authentication and have an authentication server that provides LDAP, like Authentik, manage the users. You can probably do something in Authentik to make a user temporary able to login or have an account deactivated after a date. It is a very flexible tool once you know it.

**Reply by ruiribeiro2010** [reply]
Hello,
Thanks, I'll have to look into that.

The problem is that I don't understand these matters very well, if we had a plugin for jellyfin, it would be great.

**Reply by TimGels** [reply]
LDAP is a plugin for Jellyfin. But Authentik and LDAP or not trivial things to get up and running.

**Reply by ruiribeiro2010** [reply]
ok, thanks
